Documentation/doc/CMakeLists.txt

```diff
@@ -220,10 +220,15 @@ set(CGAL_DOC_DXY_DIR "${CMAKE_BINARY_DIR}/doc_dxy")
 file(MAKE_DIRECTORY "${CGAL_DOC_DXY_DIR}")
 
 #Setting the resource directory depending on the version of doxygen
-set(CGAL_DOC_RESOURCE_DIR_DEFAULT "${CMAKE_CURRENT_LIST_DIR}/resources/1.8.14")
+set(CGAL_DOC_RESOURCE_DIR_DEFAULT "${CMAKE_CURRENT_LIST_DIR}/resources/1.8.19")
```

Comment:
Don't use 1.8.19 it had some nasty problems. Use 1.8.20.

Reply:
Do you mean even the config files are broken?

Reply:
The cmake variable only indicates what directory to use as default for the config files, not the doxygen version to pick.

Reply:
One of the problems was with config files when they were used through a pipe (as far as I know this is not the case for CGAL).

For clarity I would not mention the number 1.8.19 here but go straight to 1.8.20 so in the future people won'yt even be tempted to use 1.8.19.